README.rst

```diff
@@ -64,6 +64,13 @@ following RFC-4752, section 3.1:
 
     import winkerberos as kerberos
 
+    def get_channel_binding_struct(response):
+        # This is 'tls-server-end-point:' + certificate hash of the server
+        application_data = b'tls-server-end-point:D01402E0F16F30ED71B02B655AD71C7B0ADA73DE5FBD8134021A794FFA1EECE8'
```

Comment:
get_cbt_application_data is awesome. To bad Python's ssl module doesn't provide a way to get that information.